services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchRulesSpec.groovy

```diff
@@ -213,6 +216,56 @@ class SwitchRulesSpec extends BaseSpecification {
         ]
     }
 
+    @Unroll
+    @Tags(Tag.NEGATIVE)
+    def "Deleting switch rules by non-existing #data.description is possible but any rules are not deleted"() {
```

Comment:
IMHO, description would sound a bit simpler if rephrased to
"Attempt to delete non-existent switch rules by supplying invalid #data.description leaves all rules intact"

Reply:
Done.